src/sentry/migrations/0431_set_monitor_status_to_object_status_only.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 2.2.28 on 2023-04-28 23:49
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def set_monitor_status_to_object_status_only(apps, schema_editor):
+    Monitor = apps.get_model("sentry", "Monitor")
+
+    for checkin in RangeQuerySetWrapperWithProgressBar(
+        # 0, 1, 2, 3 are the default ObjectStatus values, status's outside of
+        # that are the extended status from the monitor
+        Monitor.objects.exclude(status__in=(0, 1, 2, 3))
```

Comment:
You're querying `Monitor` but calling the variable `checkin` is that correct, or did you want to update checkins?

Reply:
it is incorrect, thanks for catching